cypress/e2e/tests/migration/applicationinventory/analysis/custom-rules.test.ts

```diff
@@ -119,6 +119,19 @@ describe(["@tier2"], "Bug MTA-2015: Custom Rules in analyses", function () {
         exists("CUSTOM RULE for javax.* package import");
     });
 
+    it("Verify a file is not a valid XML", function () {
+        const app = new Analysis(
+            getRandomApplicationData("tackle-testapp-fileNotValidXML", {
+                sourceData: this.appData["tackle-testapp-git"],
+            }),
+            getRandomAnalysisData(this.analysisData["tackle_testapp_fileNotValidXML"])
+        );
+        Application.open();
+        applications.push(app);
+        app.create();
```

Comment:
app.create() will open the application page right?
Application.open(); is not required .

Reply:
Thanks for noticing, Done